docs/devices/EFEKTA_PWS.md

```diff
@@ -11,7 +11,7 @@ description: "Integrate your Custom devices (DiY) EFEKTA_PWS via Zigbee2MQTT wit
 
 | Model | EFEKTA_PWS  |
 | Vendor  | Custom devices (DiY)  |
-| Description | [Plant Wattering Sensor] |
+| Description | [Plant Wattering Sensor](https://github.com/smartboxchannel/Plant-Watering-Sensor-Zigbee) |
```

Comment:
Please add this link to the notes instead OR update the description in zigbee-hersdma-conveters (this part is generated based on it)